integrated_channels/xapi/statements/base.py

```diff
@@ -15,36 +15,30 @@ class EnterpriseStatement(Statement):
     Base statement for enterprise events.
     """
 
-    def get_actor(self, username, email):
+    def get_actor(self, user, user_social_auth):
         """
         Get actor for the statement.
         """
+        social_auth_uid = user_social_auth.uid if user_social_auth else ''
+        sso_id = social_auth_uid.split(':')[-1]
+        name = sso_id if sso_id else user.username
```

Comment:
I think here we should return the email address of the user and not the edX LMS username.

Reply:
@mattdrayer we are not returning email address here. We are actually returning SSO uid after striping IDP slug and if we don't SSO uid we are returning lms username. Email address of user is in mbox property below.

Reply:
Right, I just don't think it is going to be very useful to provide a third party LRS with the edX LMS username for the user -- as a matter of fact I don't think we should be sharing this information externally at all.  It would be better to fall back to the email address of the learner as the actor identifier if there is no sso_id value available.

Reply:
@mattdrayer changed code to return email address instead of LMS username.